json_database/crypto.py

```
@@ -0,0 +1,112 @@
+    if len(key) > 16:
+        key = key[0:16]
+
+    nonce, ciphertext, tag = ciphertext[:16], ciphertext[16:-16], ciphertext[-16:]
+
+    try:
+        if not isinstance(key, bytes):
+            key = bytes(key, encoding="utf-8")
+        cipher = AES.new(key, AES.MODE_GCM, nonce)
+        data = cipher.decrypt_and_verify(ciphertext, tag)
+        return decompress_payload(data)
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)